ROADMAP.md

```diff
@@ -0,0 +1,52 @@
+# Roadmap
+
+[!WARNING] This project is still in active development and not intended for production use.
+
+This page is inspired by the discussion in https://github.com/ROCm/ROCm/discussions/4276 .
+Our goal here is document the prioritized roadmap of target architectures we plan to test and eventually support as part of TheRock.
+
+## Prioritized Target Architectures
+
+The following is a list of prioritized roadmaps divided by OS (Linux/Windows) and architecture. Each individual section is its own roadmap and we will be in parallel trying to support at least one *new* architecture per section in parallel working top-to-bottom Current focus areas are in __bold__. There will be exceptions from the "top-to-bottom" ordering occassionally based on test device availability.
+
+*Note* for the purposes of the table below Sanity-Tested = "either in CI or some light form of manual QA has been performed. "Release-Ready" will mean that its supported and tested as part of our overall release process.
```

Comment:
formatting suggestions (fixing the unclosed quotes for Sanity-Tested, matching the style across each, wrapping in an alert, adding a list, emphasis via italics)

```suggestion
> [!NOTE]
> For the purposes of the table below:
> 
> * *Sanity-Tested* means "either in CI or some light form of manual QA has been performed".
> * *Release-Ready* means "it is supported and tested as part of our overall release process".
```

Preview:
![image](https://github.com/user-attachments/assets/06316511-250e-4d35-96d7-56278b4d2ae2)